cli/types/cypress.d.ts

```diff
@@ -3047,16 +3047,29 @@ declare namespace Cypress {
 
   type PickConfigOpt<T> = T extends keyof DefineDevServerConfig ? DefineDevServerConfig[T] : any
 
+  type ProjectConfig = {
```

Comment:
Let's make this type more specific, like `AngularDevServerProjectConfig`, or you can inline the type and extract it in the dev-server. `ProjectConfig` sounds like it might apply to other Cypress types